tokens/README.md

```diff
@@ -125,3 +125,115 @@ recipient address. That SOL can be used to pay transaction fees on staking
 operations such as delegating stake. The rest of the allocation is put in
 a stake account. The new stake account address is output in the transaction
 log.
+
+## Distribute SPL tokens
+
+Distributing SPL Tokens works very similarly to distributing SOL, but requires
+the `--owner` parameter to sign transactions. Each recipient account must be an
+system account that will own an Associated Token Account for the SPL Token mint.
+The Associated Token Account will be created, and funded by the fee_payer, if it
+does not already exist.
+
+Send SPL tokens to the recipients in `<RECIPIENTS_CSV>`.
+
+Example recipients.csv:
+
+```text
+recipient,amount
+CYRJWqiSjLitBAcRxPvWpgX3s5TvmN2SuRY3eEYypFvT,75.4
+C56nwrDVFpPrqwGYsTgQxv1ZraTh81H14PV4RHvZe36s,10
+7aHDubg5FBYj1SgmyBgU3ZJdtfuqYCQsJQK2pTR5JUqr,42.1
+7qQPmVAQxEQ5djPDCtiEUrxaPf8wKtLG1m6SB1brejJ1,20
+```
+
+You can check the status of the recipients before beginning a distribution. You
+must include the SPL Token mint address:
+
+```bash
+solana-tokens spl-token-balances --mint <ADDRESS> --input-csv <RECIPIENTS_CSV>
+```
+
+Example output:
+
+```text
+Token: JDte736XZ1jGUtfAS32DLpBUWBR7WGSHy1hSZ36VRQ5V
+Recipient                                             Expected Balance            Actual Balance                Difference
+CYRJWqiSjLitBAcRxPvWpgX3s5TvmN2SuRY3eEYypFvT                     75.40                      0.00                    -75.40
+C56nwrDVFpPrqwGYsTgQxv1ZraTh81H14PV4RHvZe36s                    10.000  Associated token account not yet created
+7aHDubg5FBYj1SgmyBgU3ZJdtfuqYCQsJQK2pTR5JUqr                     42.10                      0.00                    -42.10
+7qQPmVAQxEQ5djPDCtiEUrxaPf8wKtLG1m6SB1brejJ1                    20.000  Associated token account not yet created
+```
+
+To run the distribution:
+
+```bash
+solana-tokens distribute-spl-tokens --from <ADDRESS> --owner <KEYPAIR> \
+    --input-csv <RECIPIENTS_CSV> --fee-payer <KEYPAIR>
```

Comment:
Is `--fee-payer` optional?

Reply:
Right now, no, `--fee-payer` required, following the precedent in the app. But fee-payer could easily fall back to the `--owner` if desired.

Reply:
wfm, thanks

Reply:
Sorry, do you mean it works as is? Or you'd like the fallback?

Reply:
I'd prefer `--fee-payer` to fall back to `solana config get keypair`